tooling/eslint-plugin/README.md

```
@@ -0,0 +1,149 @@
+pnpm add --save-dev eslint
+```
+
+### With Yarn
+
+```sh
+yard add --dev eslint
+```
+
+Next, install `@wso2/eslint-plugin`:
+
+### With npm
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)